web/components/login/Login.js

```diff
@@ -9,11 +9,15 @@ class Login extends React.Component {
   constructor(props) {
     super(props);
 
+    // Deconstruct props and state
+    const { isLoaded, isEmpty, firebase, auth } = this.props;
+    const { email, password } = this.state;
+
     this.state = {
       email: '',
       password: '',
-      loading: !this.props.auth.isLoaded,
-      loggedIn: !this.props.auth.isEmpty,
+      loading: !isLoaded,
+      loggedIn: !isEmpty,
```

Comment:
Same here, we probably want to specifically check if `auth` is loaded, not `props`

Reply:
nice catch, ill fix this

Reply:
see my other comment, this is just adapting the previous code to the new lint rules, I want to avoid making any non-cosmetic changes

Reply:
21b0c48